configs/default/LDAR-LDARCF411.config

```diff
@@ -4,10 +4,12 @@ manufacturer_id LDAR
 
 # resources
 resource BEEPER 1 B02
-resource MOTOR 1 A00
-resource MOTOR 2 B10
+resource MOTOR 1 B04
```

Comment:
Has this board ever been sold in the existing configuration? If so, this should be added as a new target, and marketed as a separate version of the board.

Reply:
Hello mikeller. I also received a request from this manufacturer to make the above modifications. Their products should not have been sold.